src/sidebar/services/test/tags-test.js

```diff
@@ -83,6 +95,14 @@ describe('TagsService', () => {
       assert.deepEqual(tags.filter('b', 2), ['bar', 'bar argon']);
       assert.deepEqual(tags.filter('b', 3), ['bar', 'bar argon', 'banana']);
     });
+
+    it('returns non-word ASCII tags that start with the query string', () => {
+      assert.deepEqual(tags.filter('^žŸ¡\\'), ['^žŸ¡\\¢£¤']);
+    });
+
+    it('returns non-English tags that start with the query string', () => {
```

Comment:
```suggestion
    it('returns non-ASCII tags that start with the query string', () => {
```

Although this example is indeed non-English, it is the use of non-ASCII characters that matters.